README.md

```diff
@@ -126,8 +126,26 @@ Initialize the first node:
 $ swarmd -d /tmp/node-1 --listen-control-api /tmp/manager1/swarm.sock --hostname node-1
 ```
 
-In two additional terminals, join two nodes (note: replace `127.0.0.1:4242` with the address of the first node). The join token can be fetched by inspecting the cluster of the nodes (e.g. swarmctl cluster inspect default).
+Before joining cluster, the token should be fetched first:  
+
+```  
+$ export SWARM_SOCKET=/tmp/manager1/swarm.sock  
+$ swarmctl cluster inspect default  
+    ID  : atvf35yiegho7jezhjo01b964  
+    Name: default
+    Orchestration settings:
+      Task history entries: 5
+    Dispatcher settings:
+      Dispatcher heartbeat period: 5s
+    Certificate Authority settings:
+      Certificate Validity Duration: 2160h0m0s
+      Join Tokens:
+    Worker: SWMTKN-1-0r8ujt282xrdsjr2qixvotrlxupwkzt0jarx68r3f6ixq5mlby-59vee7qt3b36tj51209v1xuba
+    Manager: SWMTKN-1-0r8ujt282xrdsjr2qixvotrlxupwkzt0jarx68r3f6ixq5mlby-dmkp2w26j7x4v4q15jio2agya
 
+```
+
+In two additional terminals, join two nodes (note: replace `127.0.0.1:4242` with the address of the first node, and use the `<TOKEN>` acquired in above):
```

Comment:
Let's tell people to use the "manager" token, because it looks like the examples below are based on joining two managers.

"acquired in above" should be "acquired above".